iterators_task_1/student/src/impl.cpp

```diff
@@ -1,2 +1,62 @@
 #include "impl.h"
 
+#include <sstream>
+#include <iterator>
+#include <algorithm>
+#include <list>
+
+
+std::string reverseWords(const std::string& input)
+{
+    if (input.empty())
+    {
+        return {};
+    }
+
+    std::istringstream inputStream {input};
+    std::ostringstream outputStream;
+    std::list<std::string> container = {};
```

Comment:
@smay1613 , what about the list for this case (to avoid reallocation in the vector for long input string)?